cuequivariance_torch/tests/operations/symmetric_contraction_test.py

```diff
@@ -30,7 +32,7 @@
 @pytest.mark.parametrize("dtype", [torch.float64, torch.float32])
 @pytest.mark.parametrize("layout", [cue.ir_mul, cue.mul_ir])
 @pytest.mark.parametrize("original_mace", [True, False])
-@pytest.mark.parametrize("batch", [0, 32])
+@pytest.mark.parametrize("batch", [1, 32])
```

Comment:
We must not forget to add back this test batch=0 when the backend properly supports it